cli/config.go

```diff
@@ -42,29 +43,31 @@ var configPaths = []string{
 
 // configFlags is a mapping of cli flag names to config keys to bind.
 var configFlags = map[string]string{
-	"log-level":          "log.level",
-	"log-output":         "log.output",
-	"log-format":         "log.format",
-	"log-stacktrace":     "log.stacktrace",
-	"log-source":         "log.source",
-	"log-overrides":      "log.overrides",
-	"no-log-color":       "log.colordisabled",
-	"url":                "api.address",
-	"max-txn-retries":    "datastore.maxtxnretries",
-	"store":              "datastore.store",
-	"valuelogfilesize":   "datastore.badger.valuelogfilesize",
-	"peers":              "net.peers",
-	"p2paddr":            "net.p2paddresses",
-	"no-p2p":             "net.p2pdisabled",
-	"allowed-origins":    "api.allowed-origins",
-	"pubkeypath":         "api.pubkeypath",
-	"privkeypath":        "api.privkeypath",
-	"keyring-namespace":  "keyring.namespace",
-	"keyring-backend":    "keyring.backend",
-	"keyring-path":       "keyring.path",
-	"no-keyring":         "keyring.disabled",
-	"source-hub-address": "acp.sourceHub.address",
-	"development":        "development",
+	"log-level":           "log.level",
+	"log-output":          "log.output",
+	"log-format":          "log.format",
+	"log-stacktrace":      "log.stacktrace",
+	"log-source":          "log.source",
+	"log-overrides":       "log.overrides",
+	"no-log-color":        "log.colordisabled",
+	"url":                 "api.address",
+	"max-txn-retries":     "datastore.maxtxnretries",
+	"store":               "datastore.store",
+	"valuelogfilesize":    "datastore.badger.valuelogfilesize",
+	"peers":               "net.peers",
+	"p2paddr":             "net.p2paddresses",
+	"no-p2p":              "net.p2pdisabled",
+	"allowed-origins":     "api.allowed-origins",
+	"pubkeypath":          "api.pubkeypath",
+	"privkeypath":         "api.privkeypath",
+	"keyring-namespace":   "keyring.namespace",
+	"keyring-backend":     "keyring.backend",
+	"keyring-path":        "keyring.path",
+	"no-keyring":          "keyring.disabled",
+	"no-encryption-key":   "keyring.noencryptionkey",
+	"keyring-secret-file": "keyring.secretfile",
```

Comment:
question: What do you think about making this more general purpose and renaming to `secretfile`? If we decide to add more secrets in the future it might be confusing having the keyring name.

Reply:
I did this initially and then reverted. I think I'm more and more in favour of making it general purpose. I'll change it.

Reply:
Moved to `secretfile`.